advanced-qgis.Rmd

```diff
@@ -382,7 +382,7 @@ We will now learn how to build a model that runs the above processing steps in a
 knitr::include_graphics('images/advanced_qgis/modeler1.png')
 ```
 
-2. Launch the modeler from **Processing &rarr; Graphical Modeler** 
+2. Launch the modeler from **Processing &rarr; Model Designer** 
```

Comment:
Good catch. Let's fix all other instances of 'graphical modeler' to 'model designer' for this course in this same PR. And send a separate PR for qgis-tutorials.

Reply:
Updated.